src/App.js

```diff
@@ -1,12 +1,23 @@
 import React from 'react';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
 import Calculator from './components/Calculator';
 import Quotes from './components/Quotes';
+import Navbar from './components/Navbar';
+import Footer from './components/Footer';
+import Home from './components/Home';
 
 function App() {
   return (
     <div>
-      <Calculator />
-      <Quotes />
+      <Router>
+        <Navbar />
+        <Routes>
+          <Route path="/Home" element={<Home />} />
```

Comment:
- **[OPTIONAL]** When the app loads the footer is what gets displayed. The home component should be the root path of the app, so that when the users load the app, the home component is displayed without having to click the link on the navbar.

  ```
  <Route path="/" element={<Home />} />
  ```